mtest/libmscore/CMakeLists.txt

```diff
@@ -12,8 +12,7 @@
 #=============================================================================
 
 subdirs(
-      album barline beam breath chordsymbol clef clef_courtesy compat concertpitch copypaste
-          copypastesymbollist cursor dynamic earlymusic element exchangevoices hairpin instrumentchange join keysig layout links parts measure midi      midimapping note plugins repeat selectionfilter selectionrangedelete spanners split splitstaff timesig tools transpose tuplet text
+      album barline beam breath chordsymbol clef clef_courtesy compat concertpitch copypaste copypastesymbollist cursor dynamic earlymusic element exchangevoices hairpin instrumentchange join keysig layout links parts measure midi
```

Comment:
Hmm they should all be there and I should have added the implode_explode direction